samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -800,8 +800,10 @@ class SamzaContainer(
 
       val startTime = System.nanoTime()
       status = SamzaContainerStatus.STARTING
-
-      jmxServer = new JmxServer()
+      val isJMXEnabled = new ClusterManagerConfig(config).getJmxEnabled
+      if (isJMXEnabled) {
```

Comment:
We report virtual machine runtime metrics via `JvmMetrics` from `SamzaContainer`. Have we tested if conditionally enabling/disabling `JmxServer` has any effect on that?

Reply:
will check and circle back